packages/jest-cli/src/run_jest.js

```diff
@@ -47,41 +47,61 @@ const getNoTestsFoundMessage = (testRunData, globalConfig) => {
 
   const pluralize = (word: string, count: number, ending: string) =>
     `${count} ${word}${count === 1 ? '' : ending}`;
-  const individualResults = testRunData.map(testRun => {
-    const stats = testRun.matches.stats || {};
-    const config = testRun.context.config;
-    const statsMessage = Object.keys(stats)
-      .map(key => {
-        if (key === 'roots' && config.roots.length === 1) {
+  if (globalConfig.verbose) {
```

Comment:
this whole function is getting pretty massive!
do you mind extracting it into a separate module `get_no_test_found_message.js` and splitting up a little?

something like:
```js
// ...
const getNoTestsFoundMessage = (testRunData, globalConfig) => {
  if (globalConfig.onlyChanged) {
    return getNoTestFoundRelatedToChangedFiles(globalConfig);
  }

  return globalConfig.verbose
    ? getNoTestFoundVerbose(globalConfig, testRunData)
    : getNoTestFound(globalConfig, testRunData);
}

module.exports = getNoTestsFoundMessage;
```

Reply:
Sure thing! Commit amended.